inscriptions/ISic3098.xml

```diff
@@ -179,28 +184,27 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331423">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — — — — — — —</supplied>
-                    <lb n="2"/>
-                    <unclear>Γ</unclear>ΗΙΙ
-                    <unclear>ΑΟ</unclear>
-                    <supplied reason="lost">— — — — ὑπατίᾳ</supplied>
-                    <lb n="3"/>τοῦ δεσπ<supplied reason="lost">ότου ἡμῶν Θεο</supplied>-<lb n="4"/>δοσίου
-                    <supplied reason="lost">τὸ —ʹ κ(αὶ) — —</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><orig><unclear>Γ</unclear>ΗΙΙ<unclear>ΑΟ</unclear></orig><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">ὑπατίᾳ</supplied>
+                    <lb n="3"/>τοῦ δεσπ<supplied reason="lost">ότου</supplied> <supplied reason="lost">ἡμῶν</supplied> <supplied reason="lost">Θεο</supplied>
+                    <lb n="4" break="no"/>δοσίου <supplied reason="lost">τὸ</supplied> <supplied reason="lost"><num value="?">?</num></supplied> <supplied reason="lost"><expan><abbr>κ</abbr><ex>αὶ</ex></expan></supplied> <!-- do we want to flag here that there is a number even if we don't know its value? -->
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after SEG</note> <!-- check Orsi 1893 (p.389 missing) -->
```

Comment:
p.389 is available in the copy of NSA 1893 in the NSA folder in reference works:
![image](https://user-images.githubusercontent.com/25080386/104304473-5072f880-54c3-11eb-8372-0291147bfadc.png)